app/views/reports/regions/_header.html.erb

```diff
@@ -95,6 +92,9 @@
           </div>
         <% end %>
       </div>
+      <div class="ml-8px">
+        <%= render "downloads" %>
+      </div>
```

Comment:
Will this move where the downloads are in desktop?

Reply:
Posted a screenshot.